### ISSUE TITLE
Add term thickened cortex of the radius

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10452]
**Preferred term label:**
Thickened cortex of the radius

**Synonyms**
Broad cortex of radius- Cortical thickening of the radius- Thickened cortices of radius

**Definition (free text, please give PubMed ID)**
Abnormal thickening of the cortex of the radius
PMID24389367

**Parent term (use hpo.jax.org/app)**
Thickened cortex of long bones HP:0000935
Cortical thickening of the forearm bones HP:0003957
Abnormal morphology of the radius HP:0002818

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Caffey Disease (omim114000), for example: PMID24390061

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000811
- label: Thickened cortex of the radius
- parents: [Thickened cortex of long bones (HP:0000935)](https://hpo.jax.org/app/browse/term/HP:0000935) | [Abnormal morphology of the radius (HP:0002818)](https://hpo.jax.org/app/browse/term/HP:0002818)
- pmid: PMID:24389367|PMID:24390061
- def: Abnormally increased thickness of the cortical layer of the radius.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10452
- synonyms:
